### PR TITLE
checking jsonrpccpp version when looking for library

### PR DESCRIPTION
**Pull request by debris**
version 0.4 required

### CODE REVIEW

**Comment by debris** [comment]
looks like, these changes are not working with cmake 2.8.12. I will try to update

File: cmake/EthDependencies.cmake [hunk 1]
```diff
@@ -45,16 +45,9 @@ find_package (Jsoncpp 0.60 REQUIRED)
...
-# TODO make headless client optional
 # TODO get rid of -DETH_JSONRPC
 if (JSONRPC)
-
-	find_package (JsonRpcCpp 0.3.2)
-	if (NOT JSON_RPC_CPP_FOUND)
-		message (FATAL_ERROR "JSONRPC 0.3.2. not found")
-	endif()
+	find_package (json_rpc_cpp 0.4 EXACT REQUIRED)
 	message (" - json-rpc-cpp header: ${JSON_RPC_CPP_INCLUDE_DIRS}")
```
Comment (chfast) [inline cmake/EthDependencies.cmake]: It's not the change, but this " -" padding in massages is added by cmake when message level is STATUS:
`message(STATUS "json-rpc-cpp header")`

Comment (debris) [inline cmake/EthDependencies.cmake]: yeah, I know. I will fix it soon :)

**Comment by gavofyork** [comment]
???

**Comment by debris** [comment]
I will fix && rebase pr after I finish jsonrpc stuff (probably during the weekend).

**Comment by debris** [comment]
should be fixed, let's wait for linux && windows buildslaves to confirm this

**Pull request merged**